### PR TITLE
add blog for release v1.8.0

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <1015105054@qq.com>

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 docs update

* **What is the current behavior?** (You can also link to an open issue here)
add an English blog for KubeEdge release v1.8.0

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by gy95** [comment]
@fisherxu PTAL

File: content/en/blog/release-v1.8/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+
+### EdgeMesh Architecture Modification
+
+EdgeMesh now has two parts: edgemesh-server and edgemesh-agent. 
+The edgemesh-server requires a public IP address, when users use cross lan communication, it can act as a relay server in the LibP2P mode or assist the agent to establish p2p hole punching.
+The edgemesh-agent is used to proxy all application traffic of user nodes, acts as an agent for communication between pods at different locations.
+
+Refer to the links for more details.
+([#19](https://github.com/kubeedge/edgemesh/pull/19))
```
Comment (kevin-wangzefeng) [inline content/en/blog/release-v1.8/index.md:69]:
```suggestion
([edgemesh#19](https://github.com/kubeedge/edgemesh/pull/19))
```

File: content/en/blog/release-v1.8/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+### EdgeMesh Cross LAN Communication
+
+Users can use cross LAN communication feature to implement cross LAN edge to edge application communication and cross LAN edge to cloud application communication.
+
+Refer to the links for more details.
+([#26](https://github.com/kubeedge/edgemesh/pull/26), [#37](https://github.com/kubeedge/edgemesh/pull/37), [#57](https://github.com/kubeedge/edgemesh/pull/57))
```
Comment (kevin-wangzefeng) [inline content/en/blog/release-v1.8/index.md:77]:
```suggestion
([edgemesh#26](https://github.com/kubeedge/edgemesh/pull/26), [edgemesh#37](https://github.com/kubeedge/edgemesh/pull/37), [edgemesh#57](https://github.com/kubeedge/edgemesh/pull/57))
```

Comment (gy95) [reply content/en/blog/release-v1.8/index.md]: done

File: content/en/blog/release-v1.8/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- Handling node/*/membership/detail ([#3025](https://github.com/kubeedge/kubeedge/pull/3025))
+
+- sync the response message unconditionally ([#3014](https://github.com/kubeedge/kubeedge/pull/3014))
+
+- support default NVIDIA SMI command ([#2680](https://github.com/kubeedge/kubeedge/pull/2680))
+
+## **Future Outlook**
+
+With the release of v1.8, KubeEdge supports Active-Active HA mode deployment, which provides better scalability support for large scale clusters, supports cross LAN communication by EdgeMesh, and supports Onvif Device Mapper. Thanks to Huawei, China Unicom, Zhejiang University SEL Lab, ARM and other organizations for their contributions, as well as all community contributors for their support!
```
Comment (fisherxu) [inline content/en/blog/release-v1.8/index.md:121]:
```suggestion
With the release of v1.8, KubeEdge supports Active-Active HA mode deployment, which provides better scalability support for large scale clusters, supports cross LAN communication by EdgeMesh, and supports Onvif Device Mapper. Thanks to Huawei, China Unicom, DaoCloud, Zhejiang University SEL Lab, ARM and other organizations for their contributions, as well as all community contributors for their support!
```

Comment (gy95) [reply content/en/blog/release-v1.8/index.md]: done

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**